warehouse/integrations/bigquery/bigquery.go

```diff
@@ -225,7 +225,7 @@ func (bq *BigQuery) createTableView(ctx context.Context, tableName string, colum
 		return nil
 	}
 
-	deduplicationQuery, err := bq.deduplicationQuery(tableName, columnMap)
+	deduplicationQuery, err := bq.deduplicationQuery(tableName, columnMap, nil)
```

Comment:
Why are we passing `nil` here? Lets add a comment

Reply:
create view, we should use whatever is in the destination config.